CoffeeChat/Controller/SocialMediaViewController.swift

```diff
@@ -0,0 +1,202 @@
+//
+//  SocialMediaViewController.swift
+//  CoffeeChat
+//
+//  Created by Lucy Xu on 10/5/20.
+//  Copyright © 2020 cuappdev. All rights reserved.
+//
+
+import UIKit
+
+class SocialMediaViewController: UIViewController {
+
+    // MARK: - Private Data vars
+    private weak var delegate: OnboardingPageDelegate?
+    private let userDefaults = UserDefaults.standard
+
+    // MARK: - Private View Vars
+    private let backButton = UIButton()
+    private let disclaimerLabel = UILabel()
+    private let nextButton = UIButton()
+    private let subtitleLabel = UILabel()
+    private let titleLabel = UILabel()
+    private let skipButton = UIButton()
+    private let instagramTextField = UITextField()
+    private let facebookTextField = UITextField()
+
+    init(delegate: OnboardingPageDelegate) {
+        self.delegate = delegate
+        super.init(nibName: nil, bundle: nil)
+    }
+
+    required init?(coder: NSCoder) {
+        fatalError("init(coder:) has not been implemented")
+    }
+
+    override func viewDidLoad() {
+        super.viewDidLoad()
+
+        backButton.setImage(UIImage(named: "backArrow"), for: .normal)
+        backButton.addTarget(self, action: #selector(backButtonPressed), for: .touchUpInside)
+        view.addSubview(backButton)
+
+        titleLabel.text = "Connect your social\nmedia?"
+        titleLabel.numberOfLines = 0
+        titleLabel.textAlignment = .center
+        titleLabel.font = ._24CircularStdMedium
+        view.addSubview(titleLabel)
+
+        subtitleLabel.text = "Help your Pear contact you"
+        subtitleLabel.textColor = .greenGray
+        subtitleLabel.font = ._12CircularStdBook
+        view.addSubview(subtitleLabel)
+
+        instagramTextField.backgroundColor = .backgroundWhite
+        instagramTextField.textColor = .black
+        instagramTextField.font = ._20CircularStdBook
+        instagramTextField.placeholder = "@Instagram Handle"
```

Comment:
nitpicky design thing -- but i find it a little weird that all the words have the first letter capitalized here, but not in the facebook textfield place holder